tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_provision.c

```diff
@@ -172,6 +205,45 @@ static void test_provisioner_pb_adv_multi(void)
 	PASS();
 }
 
+static void test_provisioner_pb_adv_reprovision(void)
```

Comment:
Please add a small docstring describing the intended behavior/feature this covers. Here and above